Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -7591,7 +7591,7 @@ void HTMLMediaElement::didStopBeingFullscreenElement()
     m_changingVideoFullscreenMode = false;
 }
 
-#if ENABLE(FULLSCREEN_API)
+#if ENABLE(FULLSCREEN_API) && PLATFORM(MAC)
```

Comment:
Rather than using PLATFORM(MAC) directly, please use a more specific macro define. Something where the name explains why the condition is needed. (e.g. SUPPORTS(FOO) or ENABLE(CONSTAINT_HDR_IN...))

Reply:
Thank you for the suggestion, much more readable now!